server/protocols/jmap-integration-testing/jmap-integration-testing-common/src/test/java/org/apache/james/jmap/methods/integration/GetMessageListMethodTest.java

```diff
@@ -206,6 +211,78 @@ public void getMessageListShouldNotFilterMessagesWhenTextFilterMatchesBodyAfterT
             .body(ARGUMENTS + ".messageIds", contains(messageId));
     }
 
+    @Category(BasicFeature.class)
+    @Test
+    public void searchByFromFieldDoesntSupportUTF8FromName() throws Exception {
```

Comment:
this is supported now as far as I know?

Reply:
I cannot find an already existed similar test case like this one.

Reply:
`searchByFromFieldShouldSupportUTF8FromName`

Reply:
`searchByFromFieldShouldSupportUTF8FromName`